torch/_inductor/fx_passes/post_grad.py

```diff
@@ -882,9 +882,14 @@ def cat_noop(inputs, dim=0):
     return len(inputs) == 1
 
 
-@register_noop_decomp(aten.view)
-def view_noop(arg, size):
-    return arg.shape == size
+@register_noop_decomp(aten.view.default)
+def view_default_noop(arg, size):
+    return arg.shape == tuple(size)
+
```

Comment:
These should use `statically_known_true` here. We should not add a guard on arg[0] == size[0] if then arg[1] != size[1]. 

would be nice to have recursive api for this. 

but just 

`return len(arg.shape) == len(size) and all(statically_known_true(a == b) for zip(arg.shape, size))`

works for now.

cc @laithsakka

Reply:
apparently `statically_known_true(sym_eq(ls1,ls2))` does this - thx u @laithsakka